### DISCUSSION TITLE
Does the "Preview" option isn't enabled yet?

### DISCUSSION THREAD

**Discussion by aflaviarv** [requarks/wiki#7176] [Help / Questions] [answered]
OS: Alpine Linux 3.17.2 x64, Fargate and Docker
Wiki.js version: 2.5.298
Database engine: PostgreSQL 14.10

I was uploading some infos in a new post and wanted to make sure I was uploading the right image so I went for de Preview option.
![image](https://github.com/requarks/wiki/assets/99829226/8fb353d8-2356-44a1-a53f-570eb3424ccf)

But as seen, i doesn't looks like available for use. Is it some permission I am not giving? Is it an upcoming update?

**Comment by NGPixel** [comment, marked as answer]
This is an upcoming feature.
